.github/workflows/core_build.yml

```diff
@@ -29,10 +29,6 @@ jobs:
             compiler: clang9
             modules: without
             extra_logs: false
-          - os: ubuntu-18.04
-            compiler: clang # default in 18.04 is clang 6
-            modules: without
-            extra_logs: false
```

Comment:
please do not just remove the clang6 but **replace** it with clang7 instead

Reply:
Okay

Reply:
Done